mirascope/core/bedrock/__init__.py

```diff
@@ -0,0 +1,24 @@
+from ._call import bedrock_call
+from ._call import bedrock_call as call
+from ._utils import AssistantMessageTypeDef, BedrockMessageParam, UserMessageTypeDef
+from .call_params import BedrockCallParams
+from .call_response import BedrockCallResponse
+from .call_response_chunk import BedrockCallResponseChunk
+from .dynamic_config import BedrockDynamicConfig
+from .stream import BedrockStream
+from .tool import BedrockTool, BedrockToolConfig
+
+__all__ = [
+    "AssistantMessageTypeDef",
+    "bedrock_call",
+    "BedrockCallParams",
+    "BedrockCallResponse",
+    "BedrockCallResponseChunk",
+    "BedrockDynamicConfig",
+    "BedrockMessageParam",
```

Comment:
aha, just saw this. looking more deeply, it looks like this message param is not quite right since it won't include `BaseMessageParam` like the others.

Perhaps we can rename this one to `InternalBedrockMessageParam` and then create a new alias for `BedrockMessageParam` that matches the other providers?